packages/scandipwa/src/route/MyAccount/MyAccount.container.js

```diff
@@ -111,11 +113,13 @@ export class MyAccountContainer extends PureComponent {
     };
 
     static isTabEnabled(props, tabName) {
-        const { isWishlistEnabled } = props;
+        const { isWishlistEnabled, newsletterActive } = props;
 
         switch (tabName) {
         case MY_WISHLIST:
             return isWishlistEnabled;
+        case NEWSLETTER_SUBSCRIPTION:
+            return newsletterActive;
 
```

Comment:
Excess white line

Reply:
@IvansZuks

Reply:
Please remove white line on line 123. It should not be there